src/main/java/baseball/game/GameConstants.java

```diff
@@ -0,0 +1,5 @@
+package baseball.game;
+
+public class GameConstants {
+    public static final int SIZE_RANDOM_NUMBER = 3;
+}
```

Comment:
상수만 관리하는 클래스는 인스턴스화되지 않게 막아 두는 게 좋아 보입니다!

```suggestion
public class GameConstants {
    public static final int SIZE_RANDOM_NUMBER = 3;
    
    private GameConstants() {
        // Empty
    }
}
```

Reply:
특별한 이유가 없으면 Empty 방식이 좋은 것 같습니다.